templates/.gitignore.tpl

```diff
@@ -7,6 +7,7 @@
 
 # Editor files
 *.swp
+*.swo
```

Comment:
suggestion: I don't want this growing for every permutation of swap file extension. Although really, this should be part of the user's global gitignore. Per [GitHub's gitignore repo](https://github.com/github/gitignore/blob/fc6ce5da28a8c3480cc8a5acad050449f72a9261/Global/Vim.gitignore#L5):
```suggestion
[._]*.sw[a-p]
```